hw3/src/index.css

```diff
@@ -0,0 +1,68 @@
+:root {
+  font-family: Inter, system-ui, Avenir, Helvetica, Arial, sans-serif;
+  line-height: 1.5;
+  font-weight: 400;
+
+  color-scheme: light dark;
+  color: rgba(255, 255, 255, 0.87);
+  background-color: #242424;
+
+  font-synthesis: none;
+  text-rendering: optimizeLegibility;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+a {
```

Comment:
It doesn't really matter for a small app like this, but it's generally best practice to use classes for styling over raw tags. If, for example, you had anchors that weren't part of your navbar in this case, they would still be stylized like they were part of the navbar unless you overwrote that.

Reply:
Hello, the index.css came as an inbuilt file when I was creating the project in stackblitz. But thank you for the information.